recipes/itk/all/conanfile.py

```diff
@@ -75,10 +76,12 @@ def requirements(self):
         self.requires("hdf5/1.14.1")
         self.requires("libjpeg/9e")
         self.requires("libpng/1.6.40")
-        self.requires("libtiff/4.5.1")
+        self.requires("libtiff/4.6.0")
         self.requires("openjpeg/2.5.0")
         self.requires("onetbb/2021.9.0")
         self.requires("zlib/[>=1.2.11 <2]")
+        if self.options.with_opencv:
+            self.requires("opencv/[>=4 <5]")
```

Comment:
```suggestion
            self.requires("opencv/4.8.1")
```
Please, do not use ranged version for OpenCV. Only few references are supporting right now: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges

Reply:
That's why there are so few version ranges in conan center! Thanks